Automattic/simplenote-macos
# Search: Sortbar

### Fix
In this PR we're implementing a brand new Sort Bar, to be presented whenever we're in Search Mode.
This new Sort Bar allows users to update the Sort Mode to be applied over Search Results.

cc @eshurakov Thanks in advance!!

Ref. #626

### Test
1. Log into your account
2. Click over `View > Note Sorting > Alphabetically: Z-A`
3. Click over the top right 🔍 icon
4. Type a keyword

- [x] Verify that right after typing a search keyword the Search Sortbar shows up at the top of the Notes List
- [ ] Verify that the preselected Sort Mode matches your selection in (2)
- [x] Verify that clicking anywhere over the Sort Bar causes the app to present a Menu with Sort Mode options
- [x] Verify that changing the Search Sort Mode immediately updates the results
- [x] Verify that if you Exit / Re-Enter Search, the app will remember the last Sort Mode selected
- [x] Verify this new UI looks great in Light / Dark Mode
- [x] Verify the Note List Title does not collision with the Window Semaphore

### Release
`RELEASE-NOTES.txt` was updated in feb44a3 with:

> New Search Sort Bar #789

### Screenshots
<img width="1117" alt="Screen Shot 2021-01-05 at 3 12 33 PM" src="https://user-images.githubusercontent.com/1195260/103683127-8d416b80-4f68-11eb-95ef-824ce03f8735.png">
